src/django_idom/config.py

```diff
@@ -12,17 +12,7 @@
 IDOM_WEB_MODULES_URL = IDOM_BASE_URL + "web_module/"
 IDOM_WS_MAX_RECONNECT_DELAY = getattr(settings, "IDOM_WS_MAX_RECONNECT_DELAY", 604800)
 
-_CACHES = getattr(settings, "CACHES", {})
-if _CACHES:
-    if "idom_web_modules" in getattr(settings, "CACHES", {}):
-        IDOM_WEB_MODULE_CACHE = "idom_web_modules"
-    else:
-        IDOM_WEB_MODULE_CACHE = DEFAULT_CACHE_ALIAS
+if "idom" in getattr(settings, "CACHES", {}):
+    IDOM_CACHE = "idom"
```

Comment:
Maybe make `IDOM_CACHE = caches[IDOM_CACHE_NAME]` where `IDOM_CACHE_NAME = "idom"` so you don't need to key into `caches` everywhere you want to use it?

Reply:
Great idea! I'll implement this.